### DISCUSSION TITLE
Discord Link not inviting / going to a Mill channel.

### DISCUSSION THREAD

**Discussion by gamlerhart** [com-lihaoyi/mill#3507] [Development]
In the Docs the chat link was updated to a Discord link. However, that link opens up Discord for me and doesn't add me / invite me to any channel. Is it the correct link to  onboard people.

The link is: https://discord.com/channels/632150470000902164/940067748103487558

**Comment by lefou** [comment]
* Fixed in https://github.com/com-lihaoyi/mill/pull/3542

**Comment by rahat2134** [comment]
@lefou , link is expired again - 
<img width="1118" alt="Screenshot 2024-10-13 at 10 35 11 AM" src="https://github.com/user-attachments/assets/8c7a6f90-8b51-41c5-acdb-5ba4839036ee">

PTAL.
